src/args.rs

```diff
@@ -479,10 +480,10 @@ impl<'a> ArgMatches<'a> {
         Ok(self.str_pattern(s))
     }
 
-    /// Converts a &str pattern to a String pattern, including word
+    /// Converts a &str pattern to a String pattern, including line/word
     /// boundaries or escapes if applicable.
     fn str_pattern(&self, pat: &str) -> String {
-        let s = self.word_pattern(self.literal_pattern(pat.to_string()));
+        let s = self.line_pattern(self.word_pattern(self.literal_pattern(pat.to_string())));
```

Comment:
This line exceeds 80 cols, and it's becoming quite unwieldy anyway, so i think maybe i should change it. I wasn't sure what would be the best way to do that though.